### ISSUE TITLE
typo in perlop manpage

### ISSUE THREAD

**Issue by p5pRT** [Perl/perl5#5085]
Migrated from [rt.perl.org#8596](https://rt-archive.perl.org/perl5/Ticket/Display.html?id=8596) (status was 'resolved')

Searchable as RT8596$

**Comment by p5pRT** [comment]
### From tagunov@motor.ru

This is a bug report for perl from "Anton Tagunov" \<tagunov@&#8203;motor\.ru>\,
generated with the help of perlbug 1\.33 running under perl v5\.6\.1\.




---
There's

Used on numbers\, the bitwise operators \("&"\, "|"\, "^"\, "~"\, "\<\<"\, and ">>"\)
always produce integral results\.

in perlop\, shouldn't it be

 integral\->integer?


---


---
Flags&#8203;:
&nbsp;    category=docs
&nbsp;    severity=medium


---
Site configuration information for perl v5\.6\.1&#8203;:

Configured by anthony at Wed Oct 31 20&#8203;:29&#8203;:27 2001\.

Summary of my perl5 \(revision 5 version 6 subversion 1\) configuration&#8203;:
&nbsp;  Platform&#8203;:
&nbsp;    osname=MSWin32\, osvers=4\.0\, archname=MSWin32\-x86\-multi\-thread
&nbsp;    uname=''
&nbsp;    config\_args='undef'
&nbsp;    hint=recommended\, useposix=true\, d\_sigaction=undef
&nbsp;    usethreads=undef use5005threads=undef useithreads=define usemultiplicity=define
&nbsp;    useperlio=undef d\_sfio=undef uselargefiles=undef usesocks=undef
&nbsp;    use64bitint=undef use64bitall=undef uselongdouble=undef
&nbsp;  Compiler&#8203;:
&nbsp;    cc='cl'\, ccflags ='\-nologo \-O1 \-MD \-DNDEBUG \-DWIN32 \-D\_CONSOLE \-DNO\_STRICT \-DHAVE\_DES\_FCRYPT  \-DPERL\_IMPLICIT\_CONTEXT
\-DPERL\_IMPLICIT\_SYS \-DPERL\_MSVCRT\_READFIX'\,
&nbsp;    optimize='\-O1 \-MD \-DNDEBUG'\,
&nbsp;    cppflags='\-DWIN32'
&nbsp;    ccversion=''\, gccversion=''\, gccosandvers=''
&nbsp;    intsize=4\, longsize=4\, ptrsize=4\, doublesize=8\, byteorder=1234
&nbsp;    d\_longlong=undef\, longlongsize=8\, d\_longdbl=define\, longdblsize=10
&nbsp;    ivtype='long'\, ivsize=4\, nvtype='double'\, nvsize=8\, Off\_t='off\_t'\, lseeksize=4
&nbsp;    alignbytes=8\, usemymalloc=n\, prototype=define
&nbsp;  Linker and Libraries&#8203;:
&nbsp;    ld='link'\, ldflags ='\-nologo \-nodefaultlib \-release  \-libpath&#8203;:"C&#8203;:\\usr\\local\\lib\\CORE"  \-machine&#8203;:x86'
&nbsp;    libpth="E&#8203;:\\apps\\vc98\\sdk\.1\\Lib\\\." "E&#8203;:\\apps\\ibm\\db2p\\LIB" "C&#8203;:\\usr\\local\\lib\\CORE"
&nbsp;    libs=  oldnames\.lib kernel32\.lib user32\.lib gdi32\.lib winspool\.lib  comdlg32\.lib advapi32\.lib shell32\.lib ole32\.lib oleaut32\.lib 
netapi32\.lib uuid\.lib wsock32\.lib mpr\.lib winmm\.lib  version\.lib odbc32\.lib odbccp32\.lib msvcrt\.lib
&nbsp;    perllibs=  oldnames\.lib kernel32\.lib user32\.lib gdi32\.lib winspool\.lib  comdlg32\.lib advapi32\.lib shell32\.lib ole32\.lib oleaut32\.lib 
netapi32\.lib uuid\.lib wsock32\.lib mpr\.lib winmm\.lib  version\.lib odbc32\.lib odbccp32\.lib msvcrt\.lib
&nbsp;    libc=msvcrt\.lib\, so=dll\, useshrplib=yes\, libperl=perl56\.lib
&nbsp;  Dynamic Linking&#8203;:
&nbsp;    dlsrc=dl\_win32\.xs\, dlext=dll\, d\_dlsymun=undef\, ccdlflags=' '
&nbsp;    cccdlflags=' '\, lddlflags='\-dll \-nologo \-nodefaultlib \-release  \-libpath&#8203;:"C&#8203;:\\usr\\local\\lib\\CORE"  \-machine&#8203;:x86'

Locally applied patches&#8203;:
&nbsp;    ACTIVEPERL\_LOCAL\_PATCHES\_ENTRY



---
@&#8203;INC for perl v5\.6\.1&#8203;:
&nbsp;    C&#8203;:/usr/local/lib
&nbsp;    C&#8203;:/usr/local/site/lib
&nbsp;    \.



---
Environment for perl v5\.6\.1&#8203;:
&nbsp;    HOME=C&#8203;:\\
&nbsp;    LANG \(unset\)
&nbsp;    LANGUAGE \(unset\)
&nbsp;    LC\_ALL=EN\_US
&nbsp;    LD\_LIBRARY\_PATH \(unset\)
&nbsp;    LOGDIR \(unset\)
&nbsp;    PATH=E&#8203;:\\apps\\ibm\\vaj\\eab\\bin;C&#8203;:\\usr\\local\\bin\\;e&#8203;:\\Program
Files\\ibm\\gsk5\\lib;E&#8203;:\\APPS\\ROSE\\RATION~1\\NUTCROOT\\bin;E&#8203;:\\APPS\\ROSE\\RATION~1\\NUTCROOT\\bin\\x11;E&#8203;:\\APPS\\ROSE\\RATION~1\\NUTCROOT\\mksnt;e&#8203;:\\java\\sun\\java131\\bin;e&#8203;:\\apps\\vbroker\\jre\\Bin;e&#8203;:\\apps\\vbroker\\Bin;C&#8203;:\\WINNT\\system32;C&#8203;:\\WINNT;c&#8203;:\\program
files\\util;E&#8203;:\\apps\\CacheSys\\Bin;C&#8203;:\\Program Files\\rksupport;C&#8203;:\\WINNT\\ton\\bin;E&#8203;:\\apps\\rose\\common;E&#8203;:\\apps\\rose\\Rational
Test;E&#8203;:\\apps\\borland\\delphi\\Bin;E&#8203;:\\apps\\borland\\delphi\\Projects\\Bpl;E&#8203;:\\apps\\ibm\\IBM\\IMNNQ;E&#8203;:\\apps\\ibm\\db2p\\BIN;E&#8203;:\\apps\\ibm\\db2p\\FUNCTION;E&#8203;:\\apps\\ibm\\db2p\\SAMPLES\\REPL;E&#8203;:\\apps\\ibm\\db2p\\HELP;e&#8203;:\\apps\\ibm\\websphere\\bin
&nbsp;    PERL\_BADLANG \(unset\)
&nbsp;    SHELL \(unset\)

**Comment by p5pRT** [comment]
### From [Unknown Contact. See original ticket]

At 2002\-02\-20 07&#8203;:50&#8203;:35\, tagunov@&#8203;motor\.ru wrote \[20020219\.011\]&#8203;:
>
> in perlop\, shouldn't it be
>
>  integral\->integer?

No&#8203;:
&nbsp;    3\. \(Math\.\)
&nbsp;        \(a\) Of\, pertaining to\, or being\, a whole number or undivided
&nbsp;            quantity; not fractional\.

\(From dict\.org\)

At 2002\-02\-20 07&#8203;:49&#8203;:01\, tagunov@&#8203;motor\.ru wrote \[20020219\.012\]&#8203;:
>
> > You may mail corrections\, additions\, and suggestions to
> > perlfaq\-suggestions@&#8203;perl\.com \[\.\.\.\]
>
> I believe the perlfaq should be changed to reflect this policy

Since the perlfaq is now independently maintained\, I think the current
instructions are correct \(though the email address may need updating\)\.

At 2002\-02\-20 07&#8203;:49&#8203;:08\, tagunov@&#8203;motor\.ru wrote \[20020219\.013\]&#8203;:
>
> I beleive it should be
>   $@&#8203;\!~/unmatched/i
> not
>   $@&#8203;\!~/unmatched/
> to make it work

Thanks\, fixed\.

At 2002\-02\-20 07&#8203;:49&#8203;:15\, tagunov@&#8203;motor\.ru wrote \[20020219\.014\]&#8203;:
>
> The term 'catenation' is used in several times in the perl
> manpages\. Looks like it is used to mean 'concatenation'\.
> Is it an abbrevation?

No&#8203;:
&nbsp;    Catenation \\Cat\`e\*na"tion\\\, n\. \[L\. catenatio\.\]
&nbsp;        Connection of links or union of parts\, as in a chain; a
&nbsp;        regular or connected series\. See \{Concatenation\}\. \-\-Sir T\.
&nbsp;        Browne\.

\(Also from dict\.org\)

At 2002\-02\-20 07&#8203;:52&#8203;:50\, tagunov@&#8203;motor\.ru wrote \[20020219\.015\]&#8203;:
> 
> LINE&#8203;: while \(defined\($line = \<ARGV>\)\) \{
>     chomp\($line\);
>     if \($line =~ s/\\\\$//\) \{
>         $line \.= \<ARGV>;
>         redo LINE unless eof\(\); \# not eof\(ARGV\)\!
>     \}
>     print "$line\\n";
> \}continue\{print '\!';\}
> 
> prints '\!' only for lines that do not end with a backslash\.

In other words\, lines which were discarded by the regex\.

\- ams
